.changeset/flags-consume-resources.md

```
@@ -0,0 +1,13 @@
+---
+'foldkit': minor
+---
+
+Let the `flags` Effect require services from the `resources` Layer.
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)